app/Http/Transformers/SignupTransformer.php

```diff
@@ -26,6 +26,7 @@ class SignupTransformer extends TransformerAbstract
      */
     public function transform(Signup $signup)
     {
+        dd($signup->events);
```

Comment:
@DFurnes It looks like the Relations and Attribute Closure I made in the ModelFactory isn't quite right. It gets mad and returns this as an empty collection and breaks with `'submission_type' => $signup->events->first()->submission_type,` below.